### DISCUSSION TITLE
[Weekly Review Report]: July 20 - August 18 (since previous report)

### DISCUSSION THREAD

**Discussion by khanspers** [wikipathways/wikipathways-help#127] [curation-reports]
### Recent changes

 * Many changes to layout, ontology terms, descriptions, titles etc by @eweitz. Nice work!
 * Updates to several "Needs Work" yeast pathways

### Under construction

 * [WP5436](https://classic.wikipathways.org/index.php/Pathway:WP5436): Added deletion tag
 * Added "Needs Work" tag with suggestions to [WP5454](https://classic.wikipathways.org/index.php/Pathway:WP5454),  [WP5457](https://classic.wikipathways.org/index.php/Pathway:WP5457), [WP5459](https://classic.wikipathways.org/index.php/Pathway:WP5459), [WP5460](https://classic.wikipathways.org/index.php/Pathway:WP5460), [WP5462](https://classic.wikipathways.org/index.php/Pathway:WP5462) (all AOP pathways) and  [WP5465](https://classic.wikipathways.org/index.php/Pathway:WP5465), [WP5469](https://classic.wikipathways.org/index.php/Pathway:WP5469) 
 * Incomplete: [WP5468](https://classic.wikipathways.org/index.php/Pathway:WP5468) 
 * [WP5477](https://classic.wikipathways.org/index.php/Pathway:WP5477) and [WP5482](https://classic.wikipathways.org/index.php/Pathway:WP5482): These look almost complete. Added ontology terms and asked the author to confirm the status.
 * [WP5483](https://classic.wikipathways.org/index.php/Pathway:WP5483): contacted author

### New pathways

 * [WP5486](https://classic.wikipathways.org/index.php/Pathway:WP5486): No tags. Can @egonw confirm the status?
 * [WP5487](https://classic.wikipathways.org/index.php/Pathway:WP5487): I'm still working on it.

### Deleted Pathways

 * WP5450 was deleted based on author request.

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by egonw** [comment]
About WP5486, I think there is more knowledge and need a bit more time to figure it out. But it's a good stub. I'll check this week, as I already (when I asked for it) feedback that perhaps the enzymes can be linked to some of the conversions. That was not obvious the first time I looked at those articles, but will check again.
